packages/RichTextEditing-Core.package/TextStructure.class/instance/actOnClickFor..st

```diff
@@ -0,0 +1,4 @@
+mouse events
+actOnClickFor: model
+
+	^ (self attributes collect: [:each | each actOnClickFor: model]) includes: true
```

Comment:
I think `anySatisfy` is the right one

Reply:
This results in `(...) anySatify: [:each | each]`. This is maybe not as clear.

Reply:
I meant
```suggestion
	^ self attributes anySatisfy: [:each | each actOnClickFor: model]
```

Reply:
`anySatify` stops executing when one check returns true, but we want to execute it for all, several TextStructures could have actions associated that all should be executed.